src/main/java/org/broadinstitute/hellbender/utils/bigquery/QueryAPIRowReader.java

```diff
@@ -0,0 +1,33 @@
+package org.broadinstitute.hellbender.utils.bigquery;
+
+import com.google.cloud.bigquery.*;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.Iterator;
+
+public class QueryAPIRowReader implements Iterable<FieldValueList>, Iterator<FieldValueList> {
```

Comment:
Can you explain why this class is needed? Eg., why can't you just use the `TableResult` iterator directly instead of having this wrapper?